### ISSUE TITLE
RChain on Bounty-one Forum Report(July) 201807

### ISSUE THREAD

**Issue by jasoncruzzy** [rchain/bounties#857]
### Benefit to RChain
RChain on Bounty-one Discourse web #778  have made significant progress, It has been a great platform to showcase the RChain ideas 

 RChain Bounty-one Forum Report(July)

Last Month(june) I created RChain account on Bounty-one Forum to act as a representative(developer) for RChain, sell the RChain ideologies, create RChain awareness amongst developers and possibly on board newdevelopers for RChain.

  
**STATS FOR JULY**
1. Contents Generated for RChain on Bounty-one 

* Web interface to play with the Rholang Language~[https://talk.bountyone.io/t/web-interface-to-play-with-the-rholang-language-check-it-out-fam/223?u=rchain_coop](https://talk.bountyone.io/t/web-interface-to-play-with-the-rholang-language-check-it-out-fam/223?u=rchain_coop)

* RChain(Rholang)Scalable, Independent, Performs better~[https://talk.bountyone.io/t/rchain-rholang-scalable-independent-performs-better/267?u=rchain_coop](https://talk.bountyone.io/t/rchain-rholang-scalable-independent-performs-better/267?u=rchain_coop)

* Ready to learn Rholang~[https://talk.bountyone.io/t/ready-to-learn-rholang/283?u=rchain_coop](https://talk.bountyone.io/t/ready-to-learn-rholang/283?u=rchain_coop)

* How Scalable Blockchain Technology Can Bring Us Closer to Earth 2.0 [Sponsored]~[https://talk.bountyone.io/t/how-scalable-blockchain-technology-can-bring-us-closer-to-earth-2-0-sponsored/318?u=rchain_coop](https://talk.bountyone.io/t/how-scalable-blockchain-technology-can-bring-us-closer-to-earth-2-0-sponsored/318?u=rchain_coop)

* Introduction To The Design of Computation Calculi(Grammar)- Lucius Gregory Meredith~[https://talk.bountyone.io/t/introduction-to-the-design-of-computation-calculi-grammar-lucius-gregory-meredith/387?u=rchain_coop](https://talk.bountyone.io/t/introduction-to-the-design-of-computation-calculi-grammar-lucius-gregory-meredith/387?u=rchain_coop)

* Grammar for Rho Calculus~[https://talk.bountyone.io/t/grammar-for-rho-calculus/408?u=rchain_coop](https://talk.bountyone.io/t/grammar-for-rho-calculus/408?u=rchain_coop)

* RNode v0.5.1(New)~[https://talk.bountyone.io/t/rnode-v0-5-1-new/440?u=rchain_coop](https://talk.bountyone.io/t/rnode-v0-5-1-new/440?u=rchain_coop)

* Running RNode Server without Docker~[https://talk.bountyone.io/t/running-rnode-server-without-docker/478?u=rchain_coop](https://talk.bountyone.io/t/running-rnode-server-without-docker/478?u=rchain_coop)

* Running RNode Consensus Features~[https://talk.bountyone.io/t/running-rnode-consensus-features/529?u=rchain_coop](https://talk.bountyone.io/t/running-rnode-consensus-features/529?u=rchain_coop)

2. How Many People Read the Topics? 

We had a total number of 374 Developers reading(VIEWS) our topics in the month of July 

  

3. Have any of the developers joined RChain?

The Co-founder of Bounty-one also Co-Founder of Blockgeeks(BlockChain Tutorial) indicated interest to join and commended the RChain project, still discussing. Basically I wouldn't know exactly how many developers has joined through the forum because there is no direct template to determine such in RChain. But on the RChain Profile in Bounty-one the official RChain Web is written there.

  

4. The contents(topic) used are sourced from RChain Approved Documents hence I'm acting as a representative of the RChain Cooperative.

  

5. What is the goal of topics addressed? 

* The goal of the topics created gives an insight about the RChain Ideologies(Rholang, RNode, RSpace) and to help showcase what the RChain can offer to Blockchain Technology.

  

6. How have these topics helped built RChain Interest? 

The Bounty-one Forum has helped build the RChain interest because with its platform we have constantly created awareness and provided informations about the RChain. With over 350 developers reading(Views) and learning about RChain ideas(contents), firstly it is a good marketing point and secondly with consistency we will onboard developers for RChain.

  
  

NOTE: It is ‘Most likely’ a few developers have joined the RChain without my direct knowledge because we have the RChain official website and credible RChain information on RChain Bounty-one forum PROFILE. In most cases they(developers) might want to go direct to our Web to avoid issues like spamming/scam. Also some of the terms and conditions for the Bounty-one "Avoid public show of interest or negotiations", "Quit spamming members via personal message"

_Estimated Budget of Task:_ **$[1000]**
_Timeframe:_ **The month of July**
_Measure of completion:_ **Statistics**

### Legal

_Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws._

**Comment by jasoncruzzy** [comment]
Report statistics is from June 27--July 27

**Comment by jasoncruzzy** [comment]
@AyAyRon-P  Also for the coming months I have also considered to carefully include RAM weekly debrief as a topic included, which be a very good follow up content.

**Comment by pmoorman** [comment]
@jasoncruzzy 

In my opinion, this report should be posted on issue of #778 as just a regular status update. There is no need to create separate issues to report on a platform page that's gotten **exactly 34 views.**

Asking a $1000 budget for doing so, is in my opinion something that actively damages the collaborative and trust-based spirit of the coop. As such I've put a slashing vote on this issue.

cc @dckc @deannald @kitblake @AyAyRon-P

**Comment by jasoncruzzy** [comment]
@Pmoorman I spoke with @AyAyRon-P  as a lead guide he advised me to create a separate issue for this. @pmoorman Please this issue will be put up at as a minute for the next RAM meeting on Wednesday please try to attend this time around to further defend your reasons for a zero vote.

**Comment by pmoorman** [comment]
Oke, it's good that you checked with Aaron about it. Would love to hear his opinion here (@AyAyRon-P).

I don't plan on attending the RAM meeting, and to be honest haven't got much more to add to my reasons other than what I've already outlined above. 

- I think updating us on progress for an issues is just business-as-usual (and should thus fall under the parent issue)

- The reason for my zero vote is that the requested $1000 budget is absolutely ridiculous.

If other people disagree with me and want to voice there opinion, I'd invite them all to do so. It's probably better to do that here, rather than spend a lot of time on it in a busy meetup.

cc @deannald @dckc @kitblake (so hopefully we can hear some additional opinions)

**Comment by AyAyRon-P** [comment]
Hi @jasoncruzzy @pmoorman 

Would be glad to lend some visibility and perspective from my side. Jason reached out to me last week asking me how best to report on content he had worked on for the Bounty One Forum, which to me sounded like a good idea as a follow up. He had valid responses to my questions ranging from: reads, joins to RChain, topics, unique or sourced content, and how they built interest. 

Where I misunderstood, Jason please forgive me here, was that there was already an existing issue for the work #778 which we didn't discuss only the reporting. In my opinion and what I think could be done to resolve this is close this issue - #857, and combine your report into issue #778 and put: 

**NEW BUDGET: Follow Up Report $___**

**Comment by jasoncruzzy** [comment]
The budget reflects the work done towards creating Awareness and management of the profile as discussed with AyAyARON P specifically


$1000 reflects the input and management of the RChain Cooperative on Bounty-one (Blockchain developers forum).
Let me Restate the statistics 

* 374 views(read) in total for the month of July which translates to 374 BlockChain developers viewing our topic. 
* 10comments~With the co-founder of Bounty-one and Blockgreek interested. The Co-Founder of Bounty-one and blockgeeks is a renowned BlockChain tutor and a veteran developer, his comment of approval and indications must have prompted his followers on Bounty-one to also check out our work and ideologies.
* I wouldn't know if he has joined the RChain yet as we do not have a clear template to show such, as a professional and knowing the sensitive of my job I wouldn't go asking him.
 * We have the official RChain website there(RChain Bounty-one profile) which should be enough to direct interested members on how to join the coop. He(co founder of blockgeeks) wouldn’t find the processes too tideous as a professional.
* I(RChain Profile) just earned the highest ranking as a member on Bounty-one, stating we have played by the rules, terms and conditions, It also shows our participation, passion and commitments.(facts available)
 Contents generated are sourced from approved RChain documents.
* @pmoorman u once suggested the value of a developer at $1000. At Bounty-one we have multiple developers reading and learning our ideas, one of the topic posted last month (Web Interface To Play With The 'Rholang' Language ~check it out fam!☺) got lots of participation and thumbs up
* The evidences of RChain progress in the Bounty-one are very glaring. 
 *To state the least,Promoting RChain on Bounty-one forum at $1000 should have more value for its management and participation. 
  * I have also considered including our weekly debrief reports as a topic to constantly update developers on our progress. @patrick put each weekly report $250.
 @Pmoorman I think you should kindly reconsider!

I think the marketing team should come up with a clear template to determine those onboarding developers.

**Comment by pmoorman** [comment]
@jasoncruzzy @AyAyRon-P first things first: **I have removed my slashing vote,** because we're clearly dealing with a communications mix-up here. 

Secondly, I think we now have 2 issues open, tracking essentially the same thing (= Bounty-one engagement). I suggest that to avoid confusion, either one of the issues should be closed. I'd suggest we close this one in favor of #778, but that's your call.

Finally: I think the $1,000 for a qualified developer still holds, and if you can provide compelling evidence that one or more qualified developers have joined RChain thanks to your efforts, I think a higher budget is justified. 

I'd suggest that a good way to quantify the impact would be to show I) the username of the developer in question on Discord, and ideally just asking him to introduce himself or provide a Github or whatever so we can somewhat gauge who we've got. **In our move to value-based budget allocation, I think we should strongly favor whoever can make an explicit case for the impact they're making.** I'm happy to vote a $1,000 budget once it's clear who has joined the coop, and double that if indeed a co-founder of Blockgeeks is joining RChain. 

TL;DR: let's resolve our communications mix-up by closing one of the issues, and show us that this Blockgeeks guy is coming onboard to unlock a $2.000 vote (at least from my end)

**Comment by David405** [comment]
@AyAyron-P I don't see why there is a "need-SMART-objective" label in this issue, it is well formatted by the SMART standard, see #778 for clarification.

@pmoorman I see no justification of your voting a zero budget. The statistics clearly shows that there is value generated from maintaining a BountyOne account for RChain, I dont think the budget of $1000 is overpriced, maybe @jasoncruzzy you may need to add information regarding the efforts, time you put in to support the budget.

Also, I don't think this issue needs to be and updated from the parent issue, a collaborator should decide how best to coordinate their activity, besides it has been the norm for people handling social media accounts to create new issues at the end of the month to give a report of their activities for that month.

**Comment by AyAyRon-P** [comment]
@David405 I invite you to reread my comment again. #778 needs to be referenced in the objectives because there was no clear indication that this was a report to 778, only that it was a report. Even under 778 the objectives are not SMART they're stats about Bounty-One. 

The collaborator can decide but as a marketing guide it is our responsibility to help organize the information and with this issue having some communication breakdown it would make more sense to close this issue out and attach it to 778 as a complete project from creation of Bounty-One account all the way to the monthly report, THEN next month you can do a report only issue.

**Comment by jasoncruzzy** [comment]
@AyAyRon-P #778 has been referenced to this issue. #778 was created in the month of June and had its topics(contents) created for the month of June which I provided the stats and links for verifications on the said issue#778. The Report statistics for July began from June 27th to July 27th, So this month report should an issue of its own.

**Comment by AbnerZheng** [comment]
I also think 1000$ for the budget is ridiculous. I will vote a slashing vote.

**Comment by David405** [comment]
@AyAyRon-P this issue and #778 meet the bounty criteria of SMART, they are both Specific and Mensurable, Attainable, meaning they are also Realistic and they are time bound (Monthly). I see no reason for the needs-SMART-objectives, hence I am removing it. Also, if you look carefully #778 has its own [report](https://github.com/rchain/bounties/issues/778#issuecomment-401387622), this issue is a report for the activities in BountyOne for July and so it requires a separate issue.

@AbnerZheng @pmoorman I don't see any justification for slashing here, slashing was implemented to discourage bad behaviour, except I miss something, I  see no dishonest behaviour from @jasoncruzzy on this issue.

@pmoorman says 

> The reason for my zero vote is that the requested $1000 budget is absolutely ridiculous.

Your judgement of the budget being ridiculous is not a justification for voting zero, from @jasoncruzzy report and like I said earlier, value was generated from this, meaning effort and time was put in, this makes this work deserve a reward, you may not be inclined to vote the $1000 stipulated by @jasoncruzzy and may choose to vote a lower budget but there is no justification for voting zero.

Finally, I still end by saying **it is the duty of label guides to encourage and reward participation by creating a conducive environment for people to be creative and try out new possibilities that will positively affect the RChain project, sometimes guides should avoid being overly critical and let people give their ideas a try**

**Comment by jasoncruzzy** [comment]
I removed the marketing label and added the community building label. The community building label guide approved of this as a wonderful idea and advised the work continues.
@pmoorman kindly remove your $0 vote on this issue as it no longer relates to marketing but approved by the community building team. 

@dckc @AyAyRon-P @David405 @jimscarver

**Comment by jasoncruzzy** [comment]
![screenshot_20180731-025822](https://user-images.githubusercontent.com/37456249/43505154-e96a9796-955d-11e8-8bdd-1f163fa2878c.png)

@pmoorman here👆👆 is a proof of your request yesterday, Also Here is the link just in case you want to check it out https://talk.bountyone.io/t/web-interface-to-play-with-the-rholang-language-check-it-out-fam/223?u=rchain_coop

**Comment by dckc** [comment]
@David405 writes:
> Finally, I still end by saying it is the duty of label guides to encourage and reward participation by creating a conducive environment for people to be creative and try out new possibilities that will positively affect the RChain project, sometimes guides should avoid being overly critical and let people give their ideas a try.

Being critical doesn't prevent people from giving their ideas a try. The possibility of a financial reward creates an incentive to try things. The guides are not here to be cheerleaders.  It's the duty of the [label guides][1] "to align contributions with goals of the RChain cooperative"; that is: to vote for the 1000+ members who are not directly involved in the project. Being critical isn't fun. But it's necessary. Please don't make the job any harder than it is. The continued existence of the bounty program depends on welcoming critical feedback from guides.

[1]: https://github.com/rchain/bounties/wiki/Bounty-Task-Guides

**Comment by pmoorman** [comment]
Hey all, I thought I wrote a response this morning, but it looks that it hasn't published somehow. Weird. Below is a reproduction:

1. How is this work not marketing? The goal of the issue (as per the description) is to: `sell the RChain ideologies, create RChain awareness amongst developers and possibly on board newdevelopers for RChain.` Isn't that's _exactly_ what marketing is?!

2. It's great to see that you've provided some extra data to support the impact you're making. That's really good! However, upon further inspection there are a few things that I notice:

- Contrary to what you've stated, Benji is not a blockgeeks co-founder ([LinkedIn profile](https://www.linkedin.com/in/benji-richards/)). But he works for them, and he's a developer. So that's still a good thing.

- It appears that Bounty-one is owned by Blockgeeks, and it looks to me like Benji's job is to increase engagement on the platform ([see here](https://talk.bountyone.io/t/introduce-yourself/13)). He's enthusiastic... but _that's his job!_ I haven't seen Benji on Discord or otherwise see evidence of him actually joining and contributing to RChain.

In conclusion: I've updated my vote to a $500 budget. That's half of a full developer's value, for the above reasons. As I've said above, if anyone can present evidence that people are actually engaging with RChain outside of their own platform, I'll update my vote to $1,000 (for Benji) or to $2,000 (for a blockgeeks founder)

**Comment by pmoorman** [comment]
UPDATE: @jasoncruzzy just sent me some additional proof of Benji being enthusiastic about RChain, but has only posted it in DM to me. I thought (for the sake of transparency) that I should probably share my response here, in public.

**As a rule of thumb: don't message guides in DM, but let's keep all discussion in public, so other people can also understand based on what information decisions are being made!!**

---
[my response, in Discord]

Hey Jason,

If you present the above further evidence in the issue, that would surely make it compelling to upgrade the votes further. My recommendation would be:

1. Publish what you've posted here in DM in public on Github (so everyone can see it)

2. Go after Benji, and engage with him. Encourage him to join the coop and have him say hi in Discord (that would make it easy for me to vote $1,000 and it would make this issue an excellent example of what we'd like to see more!)

3. If possible, see if you can actually chase down that Blockgeeks founder that you spoke about. I think his name is Areer (see here: https://talk.bountyone.io/t/introduce-yourself/13). If you get him onboard, I'll vote an additional $2.000 and encourage other guides to do the same

And last thing: I would just slap the "marketing" label on, because clearly this stuff is marketing :smile:

**Comment by jasoncruzzy** [comment]
![screenshot_20180801-144919](https://user-images.githubusercontent.com/37456249/43527227-fa72f408-959d-11e8-9b8c-5c208ace82e6.png)
Here is the evidence @pmoorman mentioned earlier, Good to see we finally moving towards the right direction.
  @pmoorman Onboarding professionals takes a while, it should be done gradually and professionally as well to avoiding issues of spamming or scams. @Benji is a tutor with blockgeeks already discussing with him is already yielding positive returns as seen in the 'evidence' he has promised to share informations of RChain with fellow developers which can be termed as a multiplier effect. 

NB: These are just some of the gains RChain on Bounty-one forum has produced but the core aim of this RChain on Bounty-one forum is to help create awareness for the RChain community, help provide RChain information e.g Rholang, RNode, RSpace for develolers to work with. With all of that some developers will pick interest in RChain not just Benji probably.
@pmoorman the value of $1.000 on this issue is for the management, goals and objectives of the forum which is producing result already, Ordinarily onboarding @Benji should be a separate task with peculiar rewards but I accept it on this issue as some of the successes of the RChain community on Bounty-one forum.

**Comment by pmoorman** [comment]
I understand that onboarding people takes time, and that's fine. If he comes on board in August rather than July my budget guidelines still stand, and we'll pay it in August.

For what it's worth: my $500 vote is based on the engagement evidence you provided above (which signals "value" created for the RChain Coop), rather than the work that went into it (which is only a means to an end).
